packages/js/product-editor/src/components/variations-table/downloads-menu-item/downloads-menu-item.tsx

```diff
@@ -194,8 +194,9 @@ export function DownloadsMenuItem( {
 						</MenuItem>
 					</MenuGroup>
 					<VariationQuickUpdateMenuItem.Slot
+						name=""
 						group={ 'downloads' }
-						onChange={ onChange }
+						onChange={ ( value ) => onChange( value ) }
```

Comment:
I believe with your change in `VariationActionsMenus` types this change is not necessary.

Reply:
Fixed with 6bcf79b8cf3d057364541118c6c4fe8f3b03dc8d